### DISCUSSION TITLE
I have Ideas if you are intrested

### DISCUSSION THREAD

**Discussion by Nico53425342** [Stability-AI/StableSwarmUI#179] [Ideas]
Hello. I'm a big user of Comfy-UI, I have workflow with over 500 nodes and i was trying to find a way to simplify it for the community.
I browse inside Swarm-UI for a few hours, and i find a lot of problemes, inconvency and ideas for this project.

Tell me if you have the time to hear my ideas. :)

**Comment by mcmonkey4eva** [comment]
Sure, feel free to post in https://github.com/Stability-AI/StableSwarmUI/issues 
Bear in mind that this project is still in Alpha status and many things are planned but not yet implemented.
